tests/integration/test_integration_legacy.py

```diff
@@ -96,3 +77,22 @@ def test_integration_legacy_update_set_json(service_legacy, metadata_legacy):
         gnmi.proto.legacy.GetRequest(path=[path],), metadata=metadata_legacy,
     )
     validate_response_get(response=response, value=config)
+
+
+def test_integration_legacy_delete(service_legacy, metadata_legacy):
+    path = _create_path("system/config/hostname")
+
+    service_legacy.Set(
+        gnmi.proto.legacy.SetRequest(delete=[path]), metadata=metadata_legacy
+    )
+
+    response = service_legacy.Get(
+        gnmi.proto.legacy.GetRequest(path=[_create_path("system/config")],),
+        metadata=metadata_legacy,
+    )
+    validate_response_does_not_contain(response=response, value="hostname")
+
+
+def _create_path(path) -> gnmi.proto.legacy.Path:
```

Comment:
This should probably move into `tests.integration.path` as `create_path_from_string`.

Reply:
Yeah, is was there in between. But since I have to methods for legacy and non legacy, I moved it to the test classes. Any idea if we could merge this methods?

Reply:
Pass the module as an argument maybe?

Reply:
I tried that. But I find it clearer to have two methods, its not much of duplication IMHO.